### ISSUE TITLE
Workshop review - In Exercise 1, why do they log in to the kube dashboard?

### ISSUE THREAD

**Issue by artberger** [IBM/istio101#44]
I was wondering why users log in to the kube dashboard in exercise 1. They don't do anything here.

Perhaps if they have to use the kube dashboard later in the workshop, it makes sense to tell them how to do it here. However, maybe we could shorten this to just use the GUI launch kube dashboard button instead of the older, hard way?

Similarly, why do we tell them to list their clusters and worker nodes? What do they do with this information?

**Comment by linsun** [comment]
it is just to show them the dashboard is available and teach them how to get to it.  re cluster node and worker node, it is to teach them the commands to retrieve such info so they can know more about the cluster.

**Comment by linsun** [comment]
Feel free to suggest better wordings via a PR :)

**Comment by nilesh32** [comment]
@artberger Did you submit PR for these changes?
